### DISCUSSION TITLE
Next release date ?

### DISCUSSION THREAD

**Discussion by emyes** [logfellow/logstash-logback-encoder#737] [Q&A] [answered]
Hi Team,

Good day & Happy New Year...!

Can someone please advise when is the next release planned ? I can see few commits in main branch related to logback version bumping.   I would like to know about the release version and release date for those changes. .

**Comment by brenuart** [comment, marked as answer]
Thanks, best wishes too.

There is no date scheduled yet for the next release I'm afraid...
However, the latest version (`7.0.1`) is compatible with the latest Logback releases. Simply upgrade the Logback version in your pom and you are all set ;-)

**Reply by emyes** [reply]
Thanks @brenuart . Appreciated !!

**Comment by PascalSchumacher** [comment]
A release with https://github.com/logfellow/logstash-logback-encoder/pull/711 would be great! Imho this improvement alone is enought for a new minor version, but of course it is your choice. 😃 

Thank you very much for developing logstash-logback-encoder! 👍

**Reply by brenuart** [reply]
Soon... no actual date scheduled yet but probably in the next couple of weeks.
Do you have any urgent need for this feature?

**Reply by PascalSchumacher** [reply]
Thank, that is great news! 👍  I do not have an urgent need for this feature, it's just a nice performance improvement. 😃
